### PR TITLE
Add hero

### PR DESCRIPTION
**Pull request by AnastasiiaSorina**

### CODE REVIEW

**Review by TaniaZaslon — CHANGES_REQUESTED** [review]
Let's compress image to have less weight. You can use https://imagecompressor.com/

File: src/components/hero/SkillsGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  skills: string[];
+}
+const SkillsGroup: FunctionComponent<ISkillsGroupProps> = ({
+  title,
+  skills,
+}) => {
+  return (
+    <div>
+      <h2>{title}</h2>
+      <ul className="ball">
```
Comment (TaniaZaslon) [inline src/components/hero/SkillsGroup.tsx]: We can use more descriptive class name `<ul className="ball">` => **`<ul className="skills-list">`**

**Review by TaniaZaslon — CHANGES_REQUESTED** [review]

**Comment by AnastasiiaSorina** [comment]
> Let's compress image to have less weight. You can use https://imagecompressor.com/
- [x] Done[b6368ae](https://github.com/AnastasiiaSorina/Portfolio-Project/pull/7/commits/b6368aed442f98dde3d0909a9d7da7293129bd21)

File: src/components/hero/SkillsGroup.tsx (hunk 1 above)
Comment (AnastasiiaSorina) [reply src/components/hero/SkillsGroup.tsx]: - [x] Done [04d1f5f](https://github.com/AnastasiiaSorina/Portfolio-Project/pull/7/commits/04d1f5f7e16e9493e0d94389678547fa32833269)

**Review by TaniaZaslon — APPROVED** [review]
Looks good 👍

**Pull request merged**